lib/MCT.pm

```diff
@@ -6,12 +6,15 @@ our $VERSION = '0.01';
 
 use Mojo::Pg;
 use MCT::Model;
+use MCT::Model::Countries;
 
 has pg => sub { Mojo::Pg->new(shift->config->{db}) };
 
 sub migrations {
   my $app = shift;
-  $app->pg->migrations->from_file($app->home->rel_file('mct.sql'));
+  $app->pg->migrations->from_string(
+    $app->build_controller->render_to_string(template => 'sql/migrations', format => 'sql')
+  );
```

Comment:
now that we aren't interpolating all those countries, can we go back to a simple file rather than a template?